user-access-public-model/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'nva.identity.service.java-library-conventions'
+    id 'nva.identity.service.dynamodblocal'
```

Comment:
added the plugin because the testing package depends on the dynamodb-local package